launcher/firstLaunch/firstlaunch_moc.cpp

```diff
@@ -295,13 +295,23 @@ bool FirstLaunchView::heroesDataDetect()
 QString FirstLaunchView::getHeroesInstallDir()
 {
 #ifdef VCMI_WINDOWS
-	QString gogPath = QSettings("HKEY_LOCAL_MACHINE\\SOFTWARE\\GOG.com\\Games\\1207658787", QSettings::NativeFormat).value("path").toString();
-	if(!gogPath.isEmpty())
-		return gogPath;
+	QHash<QString, QString> regKeys = {
```

Comment:
Perhaps QVector<QPair<>> is more suitable here?
It is not actually a map / dictionary, but a collection of pairs of strings - path in registry and name of registry key.

Reply:
Yes `QVector<QPair<...>>` is much better here. :)